Library/Formula/pcap_dnsproxy.rb

```diff
@@ -0,0 +1,51 @@
+class PcapDnsproxy < Formula
+  desc "A powerful DNS proxy designed to anti DNS spoofing"
+  homepage "https://github.com/chengr28/Pcap_DNSProxy"
+  url "https://github.com/chengr28/Pcap_DNSProxy/archive/v0.4.2.tar.gz"
+  sha256 "0737b16283e84bc94f63066f448ba84416141b8d5a86b5e01f1f97164d9d44f2"
+  head "https://github.com/chengr28/Pcap_DNSProxy.git"
+
+  depends_on :xcode => :build
+  depends_on "libsodium" => :build
+
+  def install
+    ln_s "#{Formula["libsodium"].opt_lib}/libsodium.a", "#{buildpath}/Source/LibSodium/LibSodium_Mac.a"
```

Comment:
Why does he vendor the headers? If it's using those headers and not the headers of our formula this may break when `libsodium` is upgraded.